.ddqa/config.toml

```diff
@@ -0,0 +1,16 @@
+global_config_source = "aHR0cHM6Ly9yYXcuZ2l0aHVidXNlcmNvbnRlbnQuY29tL0RhdGFEb2cvZ2l0aHViLW1ldGFkYXRhL21hc3Rlci9qaXJhLnRvbWw="
+jira_statuses = [
+  "TODO",
+  "Testing",
+  "Done",
+]
```

Comment:
If my board contains more statuses?

Reply:
I believe statuses mentioned here would actually be 'labels' from the JIRA perspective, as we don't want to force 20+ projects on their workflows and board organisation. @ofek can provide more implementation details.

Reply:
Yes that is correct, for now we will only have a workflow consisting of 3 stages